alpine/generate-dockerfile.sh

```diff
@@ -58,8 +58,9 @@ for repo in $repos; do
 
 		sed -e s~#{FROM}~"$baseImage:$suite"~g \
 			-e s~#{LABEL}~"$label"~g \
-			-e s~#{QEMU}~"$qemu"~g Dockerfile.tpl > $dockerfilePath/Dockerfile
-		cp entry.sh resin rc.conf resin-xbuild $dockerfilePath/
+			-e s~#{QEMU}~"$qemu"~g \
+			-e s~#{PACKAGES}~"$packages"~g Dockerfile.tpl > $dockerfilePath/Dockerfile
```

Comment:
Tini is available from Alpine 3.4 so we only install it for Alpine 3.4 and higher but we dropped support for old versions so It can be removed. I removed it.